resources/views/partials/forms/edit/company-select.blade.php

```diff
@@ -4,8 +4,8 @@
     <div class="form-group">
     {{ Form::label($fieldname, $translated_name, array('class' => 'col-md-3 control-label')) }}
         <div class="col-md-6">
-            <select class="js-data-ajax"  disabled="true" data-endpoint="companies" data-placeholder="{{ trans('general.select_company') }}" name="{{ $fieldname }}" style="width: 100%" id="company_select" aria-label="{{ $fieldname }}">
-                @if ($company_id = old($fieldname, (isset($item)) ? $item->{$fieldname} : ''))
+                 <select class="js-data-ajax"  disabled="true" data-endpoint="companies" data-placeholder="{{ trans('general.select_company') }}" name="{{ $fieldname }}" style="width: 100%" id="company_select" aria-label="{{ $fieldname }}"  {{$multiple='true' ? 'multiple': ''}}>
+                     @if ($company_id = old($fieldname, (isset($item)) ? $item->{$fieldname} : ''))
```

Comment:
I think this will always make the select box `multiple` because we're assigning `$multiple` to true by using one `=`. I _think_ we have to do something like `@if(isset($multiple) && $multiple)` to see that it is provided and true.